test/e2e/storage/subpath.go

```diff
@@ -49,8 +49,9 @@ var (
 )
 
 type volInfo struct {
-	source *v1.VolumeSource
-	node   string
+	source                    *v1.VolumeSource
```

Comment:
Please split the code changes (everything *except* making this a conformance test) into their own PR and merge that.  Then this PR should just be the addition of the conformance test labels, and I'll accept and merge that PR.

Reply:
let me split it and work that out, thanks!